web/urls.py

```diff
@@ -60,6 +60,11 @@
         views.ExperimentProxyView.as_view(),
         name="experiment-proxy",
     ),
+    path(
+        "studies/jspsych/<int:pk>/<int:child_id>/",
+        views.JsPsychExperimentView.as_view(),
+        name="jspsych-experiment",
+    ),
```

Comment:
Same question as with the preview URL structure. My preference would be to not include the experiment type in the study URL, which I think would mean we'd need an ExperimentRedirect view? I also have concerns about using the PKs in the URL since they are guessable and are probably still valid with typos.

Reply:
FYI, I added an issue about different URL formats based on experiment type: #1325